packages/types/src/clerk.ts

```diff
@@ -890,9 +890,10 @@ export type SetActiveParams = {
 
 export type SetActive = (params: SetActiveParams) => Promise<void>;
 
-export type RoutingOptions =
-  | { path: string | undefined; routing?: Extract<RoutingStrategy, 'path'> }
-  | { path?: never; routing?: Extract<RoutingStrategy, 'hash' | 'virtual'> };
```

Comment:
Easing the type safety here since we have runtime validation for invalid routing strategies.

![Screenshot 2025-01-22 at 4 01 31 PM](https://github.com/user-attachments/assets/e23ca529-34e4-48ee-a5cc-90d605dfa8c6)